### PR TITLE
Feat:"Added Cards"

### PR DESCRIPTION
**Pull request by sadathshariff**
Have added the following cards
1. Card with Badge
2. Card with Dismiss icon
3. Card with Shadow
4. Card with Text overlay
5. Card with only Text 
6. Horizontal Card
Animation

### CODE REVIEW

**Review by tanishq20 — COMMENTED** [review]
Good to merge.

File: CSS/Components/cards.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  padding: 0.3rem;
+  width: 6rem;
+  background: var(--danger-color);
+  color: white;
+  text-align: center;
+  font-weight: 600;
+  box-shadow: 4px 4px 10px rgba(109, 129, 226, 0.5);
+}
+.card-shadow {
+  box-shadow: rgb(0 0 0 / 74%) 0px 4px 8px;
```
Comment (aishwarya-bm) [inline CSS/Components/cards.css:76]: Maybe use a variable for box shadow color, it can be reused in multiple places. Other than that it's good to merge.

Comment (sadathshariff) [reply CSS/Components/cards.css:76]: Thanks, Aishwarya will do that for upcoming Components, this was particularly for this

**Comment by sadathshariff** [comment]
> Good to merge.

Thanks, Tanishq

**Pull request merged**